core/domain/config_domain.py

```diff
@@ -323,7 +323,7 @@ def get_all_config_property_names(cls):
 TOPIC_IDS_FOR_CLASSROOM_PAGES = ConfigProperty(
     'topic_ids_for_classroom_pages', SET_OF_CLASSROOM_DICTS_SCHEMA,
     'The set of topic IDs for each classroom page.', [{
-        'name': 'Math',
+        'name': 'math',
```

Comment:
Is this a user-facing text? (If yes then it should be capitalized?)

Reply:
This text would get displayed in the page title (in the browser tab) and in the header (in the content), and I've made sure that the first letter is capitalized in both instances. I made this change in order to maintain consistency since everywhere else, the classroom name is 'math'.